### ISSUE TITLE
Adjust post navigation aria-label

### ISSUE THREAD

**Issue by nielslange** [WordPress/twentytwenty#309]
As mentioned in #300, currently the `aria-label` in https://github.com/WordPress/twentytwenty/blob/ef21a6df7125a84dcfdcf61bdd4119d6a80331ef/template-parts/navigation.php#L25 is set to `Post` but should be `Post navigation` to be more descriptive.

cc: @afercia

**Comment by afercia** [comment]
Please don't 🙂 Screen readers already announce the `<nav>` element type as "navigation". The aria-label should not contain the word "navigation" again, to avoid it gets repeated unnecessarily:

<img width="1310" alt="Screenshot 2019-09-16 at 18 27 27" src="https://user-images.githubusercontent.com/1682452/64975634-c7888480-d8af-11e9-8309-447d36b6662c.png">

**Comment by nielslange** [comment]
Closed based on https://github.com/WordPress/twentytwenty/issues/309#issuecomment-531854465
